opencti-platform/opencti-graphql/tests/02-integration/02-resolvers/ingestion-csv-test.ts

```diff
@@ -57,8 +58,39 @@ describe('CSV ingestion resolver standard behavior', () => {
       `,
       variables: CSV_FEED_INGESTER_TO_CREATE
     });
-    singleColumnCsvFeedIngesterId = createSingleColumnCsvFeedsIngesterQueryResult?.data?.ingestionCsvAdd?.id;
+    singleColumnCsvFeedIngesterId = createSingleColumnCsvFeedsWithInlineMapperIngesterQueryResult?.data?.ingestionCsvAdd?.id;
     expect(singleColumnCsvFeedIngesterId).toBeDefined();
+    expect(createSingleColumnCsvFeedsWithInlineMapperIngesterQueryResult?.data?.ingestionCsvAdd?.entity_type).toBe('IngestionCsv');
+    expect(createSingleColumnCsvFeedsWithInlineMapperIngesterQueryResult?.data?.ingestionCsvAdd?.ingestion_running).toBeFalsy();
+  });
+
+  it('should create a CSV feeds ingester with inline CSV Mapper', async () => {
+    const CSV_FEED_INGESTER_TO_CREATE = {
+      input: {
+        authentication_type: 'none',
+        name: 'Single column',
+        uri: 'https://lists.blocklist.de/lists/all.txt',
+        csv_mapper: JSON.stringify(singleColumnCsvMapper),
+        csv_mapper_type: 'inline',
+        user_id: ADMIN_USER.id
+      }
+    };
+    const createSingleColumnCsvFeedsIngesterQueryResult = await queryAsAdmin({
```

Comment:
You can eventually use queryAsUserWithSuccess https://github.com/OpenCTI-Platform/opencti/blob/1223014c228ecab766e3931be62ff1651dda23c0/opencti-platform/opencti-graphql/tests/utils/testQueryHelper.ts#L85

With user USER_DISINFORMATION_ANALYST (it's this only one in test user that has ingestion capability) instead of admin query.